libs/ui-toolkit/src/components/async-renderer/async-renderer.tsx

```diff
@@ -26,13 +27,16 @@ export function AsyncRenderer<T = object>({
   render,
 }: AsyncRendererProps<T>) {
   if (error) {
-    return (
-      <Splash>
-        {errorMessage
-          ? errorMessage
-          : t(`Something went wrong: ${error.message}`)}
-      </Splash>
-    );
+    Sentry.captureException(`Error rendering data: ${error.message}`);
+    if (!data) {
+      return (
+        <Splash>
+          {errorMessage
+            ? errorMessage
+            : t(`Something went wrong: ${error.message}`)}
```

Comment:
(non-blocking): I think displaying error.message is not necessary. Something went wrong would be enough here

Reply:
I think we should still have the error message out there. People testing will see what is wrong more easily.